ground/gcs/src/plugins/uavtalk/telemetry.cpp

```diff
@@ -377,23 +377,21 @@ void Telemetry::transactionTimeout(ObjectTransactionInfo *transInfo)
     transInfo->timer->stop();
     // Check if more retries are pending
     if (transInfo->retriesRemaining > 0) {
-        TELEMETRY_QXTLOG_DEBUG(
-            QString("[telemetry.cpp] Transaction timeout:%0 Instance:%1 Retrying")
+        qDebug() << QString("[telemetry.cpp] Transaction timeout:%0 Instance:%1 Retrying")
```

Comment:
Will promote all of these.  We don't **really** use levels now but it would be nice to in the future.